server/sonar-server/src/main/java/org/sonar/server/qualitygate/ws/CreateAction.java

```diff
@@ -62,17 +66,15 @@ public void define(WebService.NewController controller) {
 
   @Override
   public void handle(Request request, Response response) {
-    userSession.checkPermission(GlobalPermissions.QUALITY_GATE_ADMIN);
-    DbSession dbSession = dbClient.openSession(false);
-    try {
+    userSession.checkLoggedIn().checkOrganizationPermission(defaultOrganizationProvider.get().getUuid(), GlobalPermissions.QUALITY_GATE_ADMIN);
```

Comment:
technically, "Anyone" could be given the global perm "QUALITY_GATE_ADMIN".
Isn't this unnecessarily changing the behavior of api/quality_gates/create?